aviary/utils/test/test_fortran_to_aviary.py

```diff
@@ -57,6 +58,11 @@ def compare_files(self, filepath, skip_list=[]):
             for line in f_in:
                 if any(s in line for s in skip_list):
                     break
+                # skip first header line, as it changes based on time of execution and
+                # username
+                timestamp = datetime.now().strftime('%m/%d/%y at')
+                if f'# created {timestamp}' in line:
+                    break
```

Comment:
instead of adding this, could we just use the existing skip_list, like in `test_engine_deck_conversion.py`:
`self.compare_files(filename, skip_list=['# created'])`